languages/elixir/exercises/concept/anonymous-functions/.docs/introduction.md

```diff
@@ -0,0 +1,37 @@
+Functions are treated as first class citizens in Elixir. This means functions:
+
+- Can be assigned to variables.
+- Can be passed around like data as arguments and return values.
+- Can be created dynamically.
+
+Anonymous functions, in contrast to named functions, don't have a static reference available to them, they are only available if they are assigned to a variable or immediately invoked.
+
+We might use anonymous functions to:
+
+- Partially apply a function (also known as [currying a function][currying]).
```

Comment:
Just mentioning partial application without explaining what it is feels a bit unsatisfactory to me. So either we explain it here (not my preference) or else not mention it and explain partial application in a different exercise (my preference). A partial application could also nicely introduce the `|>` operator.